site.js

```diff
@@ -0,0 +1,77 @@
+// eslint-disable-next-line no-undef
+const { DateTime } = luxon;
```

Comment:
[ Optional Changes ]

- [ ] Let's try not to disable Linter checks in our codebase as Linter checks are there to help us write clean, correct, and professional code. Kindly solve linter errors and have your codebase professional and error-free from Linter cheks 👍🏽